en/docs/design-your-project/index.md

```diff
@@ -1,4 +1,4 @@
-# Architecture Diagram
+# Design your project
```

Comment:
We don't need to say `(i.e., formerly, the Project Design View)`

Reply:
Removed.